diff --git a/src/defaults.py.old b/src/defaults.py
index 37e3b7a..2dcdd86 100644
--- a/src/defaults.py.old
+++ b/src/defaults.py
@@ -61,8 +61,10 @@ system = {
     'ENABLED': True,
     'KERNEL_CMD': '/proc/cmdline',
     'SET_CLOCK_CMD': '/sbin/hwclock --systohc --utc',
-    'UPDATE_REQUEST_URL': 'https://update.coreelec.org/updates.php',
-    'UPDATE_DOWNLOAD_URL': 'https://%s/%s/%s',
+    #'UPDATE_REQUEST_URL': 'https://update.coreelec.org/updates.php',
+    'UPDATE_REQUEST_URL': 'https://javatmn.us.to/ce/updates.php',
+    #'UPDATE_DOWNLOAD_URL': 'https://%s/%s/%s',
+    'UPDATE_DOWNLOAD_URL': 'https://%s.javatmn.us.to/%s/%s',
     'LOCAL_UPDATE_DIR': '/storage/.update/',
     'XBMC_RESET_FILE': '%s/reset_xbmc' % CONFIG_CACHE,
     'COREELEC_RESET_FILE': '%s/reset_oe' % CONFIG_CACHE,
